R/xcmsSet.R

```diff
@@ -1682,15 +1682,9 @@ setMethod("getEIC", "xcmsSet", function(object, mzrange, rtrange = 200,
 
         cat(sampleidx[i], "")
         flush.console()
-        lcraw <- xcmsRaw(files[sampidx[i]], profmethod = prof$method, profstep = 0)
-        if(length(object@dataCorrection) > 1){
-            if(object@dataCorrection[sampidx[i]] == 1)
-                lcraw<-stitch(lcraw, AutoLockMass(lcraw))
-        }
-        if (rt == "corrected")
-            lcraw@scantime <- object@rt$corrected[[sampidx[i]]]
-        if (length(prof) > 2)
-            lcraw@profparam <- prof[seq(3, length(prof))]
+        ## getXcmsRaw takes care of rt correction, susetting to scanrage and other
+        ## stuff.
+        lcraw <- getXcmsRaw(object, sampleidx=i, rt=rt)
```

Comment:
hi @hpbenton could you please check out that the next xcms in the devel branch after merging Jo's patch still handles the weird Waters lockmass things ? If not, this should be handles properly at the getXcmsRaw -level.
Thanks, Steffen